lib/src/native/realm_core.dart

```diff
@@ -70,7 +70,7 @@ class _RealmCore {
     return _instance ??= _RealmCore._();
   }
 
-  String get libraryVersion => _realmLib.realm_get_library_version().cast<Utf8>().toDartString();
+  String get libraryVersion => _realmLib.realm_get_library_version().cast<Utf8>().toRealmDartString()!;
```

Comment:
why not use `toDartString` here

Reply:
and on all places where we use `!` in `toRealmDartString()!`

Reply:
My thinking was that it's better to have a single method for accessing strings. That way it's less cognitive load and we'll have an easier time refactoring stuff if necessary. Getting used to using `toRealmDartString` will also remind people to check whether they need to free the native string.